### PR TITLE
Add a link to sample-controller repository on Controllers page

### PR DESCRIPTION
**Pull request by able8**
The [sample-controller](https://github.com/kubernetes/sample-controller) repository would be helpful if you want to write your own controller.

See [Preview](https://deploy-preview-43774--kubernetes-io-main-staging.netlify.app/docs/concepts/architecture/controller/)

### CODE REVIEW

**Review by kumarankit999 — COMMENTED** [review]
Looks Good to me here, but by the way, is this PR is related to any issue which has triage accepted label?

File: content/en/docs/concepts/architecture/controller.md [hunk 1]
```diff
@@ -166,5 +166,5 @@ controller does.
 * Learn more about the [Kubernetes API](/docs/concepts/overview/kubernetes-api/)
 * If you want to write your own controller, see
   [Extension Patterns](/docs/concepts/extend-kubernetes/#extension-patterns)
-  in Extending Kubernetes.
+  in Extending Kubernetes and [sample-controller](https://github.com/kubernetes/sample-controller) repository.
```
Comment (tengqm) [inline content/en/docs/concepts/architecture/controller.md:169]: maybe we can remove "in Extending Kubernetes" safely?
It doesn't add any value and the uppercase "Extending" looks weird.

Comment (tengqm) [reply content/en/docs/concepts/architecture/controller.md:169]: "and" -> "and the" ?

Comment (Ritikaa96) [reply content/en/docs/concepts/architecture/controller.md:169]: Only writing Extension patterns also looks incomplete. How about this:
`Learn how to write your own controller with 
  [Kubernetes extension patterns](/docs/concepts/extend-kubernetes/#extension-patterns)  
 and the [sample-controller](https://github.com/kubernetes/sample-controller) repository.`

Comment (able8) [reply content/en/docs/concepts/architecture/controller.md]: Updated. Thanks. @Ritikaa96  @tengqm

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Pull request merged**